sqlglot/dialects/oracle.py

```diff
@@ -205,6 +206,64 @@ def _parse_json_array(self, expr_type: t.Type[E], **kwargs) -> E:
             )
 
         def _parse_hint(self) -> t.Optional[exp.Hint]:
+            start_index = self._index
+            should_fallback_to_string = False
+
+            if not self._match(TokenType.HINT):
+                return None
+
+            hints = []
+
+            try:
+                for hint in iter(
+                    lambda: self._parse_csv(
+                        lambda: self._parse_hint_function_call() or self._parse_var(upper=True),
+                    ),
+                    [],
+                ):
+                    hints.extend(hint)
+            except ParseError:
+                should_fallback_to_string = True
+
+            if not self._match_pair(TokenType.STAR, TokenType.SLASH):
+                should_fallback_to_string = True
+
+            if should_fallback_to_string:
+                self._retreat(start_index)
+                return self._parse_hint_fallback_to_string()
+
+            return self.expression(exp.Hint, expressions=hints)
+
+        def _parse_hint_function_call(self) -> t.Optional[exp.Expression]:
+            if not self._curr:
+                return None
+
+            this = self._curr.text
+
+            if not self._next or self._next.token_type != TokenType.L_PAREN:
+                return None
+
+            self._advance(2)
+
+            args = self._parse_hint_args()
+
+            this = self.expression(exp.Anonymous, this=this, expressions=args)
+
+            self._match_r_paren(this)
+
+            return this
```

Comment:
Let's reduce the spacing here a bit, it's a bit too much.
```suggestion
            self._advance(2)
            this = self.expression(exp.Anonymous, this=this, expressions=self._parse_hint_args())
            self._match_r_paren(this)
            return this
```

Reply:
@georgesittas  please check the latest commit. I believe I have the spacing correct - just let me know if you prefer a change.